libtest.sh

```diff
@@ -0,0 +1,356 @@
+#!/usr/bin/env bash
+
+# This is a library that helps making integration tests for
+# CLI tools that talk to conodes. It is documented in doc/libtest.md.
+
+# This was copied from github.com/onet/app/libtest.sh and
+# put here because it is better to have Cothority as self-contained
+# as possible for reliable testing.
+
+# highest number of servers and clients
+NBR=${NBR:-3}
+# Per default, have $NBR servers
+NBR_SERVERS=${NBR_SERVERS:-$NBR}
+# Per default, keep one inactive server
+NBR_SERVERS_GROUP=${NBR_SERVERS_GROUP:-$(( NBR_SERVERS - 1))}
+# Show the output of the commands: 0=none, 1=test-names, 2=all
+DBG_TEST=${DBG_TEST:-0}
+# DBG-level for server
+DBG_SRV=${DBG_SRV:-0}
+# APPDIR is usually where the test.sh-script is located
+APPDIR=${APPDIR:-$(pwd)}
+# The app is the name of the builddir
+APP=${APP:-$(basename $APPDIR)}
+# Name of conode-log
+COLOG=conode
+
+RUNOUT=$( mktemp )
+
+startTest(){
+  set +m
+  if [ "$CLEANBUILD" ]; then
+    rm -f conode $APP
+  fi
+  build $APPDIR
+}
+
+test(){
+  echo "** DEPRECATED: please change to using runTest instead."
+  exit 1
+}
```

Comment:
I think it's best to just delete it. Keeping it still conflicts with the bash `test` command, which is what we're trying to fix.